src/vm/method.hpp

```diff
@@ -3277,15 +3316,33 @@ class InstantiatedMethodDesc : public MethodDesc
     }
 #endif // FEATURE_COMINTEROP
 
+    // In DAC builds this function is meant to be called only for structures, which are not part of other structures
```

Comment:
I do not think that the comment is necessary. There are probably hundreds of existing methods that make this assumption. Having the comment on a few is not very useful.